README.md

```diff
@@ -4,6 +4,8 @@ This is the [website](https://japan-grid-carbon.vercel.app/) built to show off a
 
 Its built mostly using `React` in `Typescript` but also uses `Recharts` & `i18next`
 
+> Note: You would need a Google API Key (free quota is available) to use geolocation based functionality in this application. Create a `.env` file with `REACT_APP_GOOGLE_API_KEY=<Your-Google-API-key>` in that.
+
```

Comment:
Great work!
However, since this is intended to only be a static client-side site. Spinning up a server/lambda and paying for the geolocation doesn't seem worth it.
Since its only being used to figure out the user's country. I think we'll just stick to grabbing the closest Utility and forget the country bit 👍 
Please remove the `fetchCurrentUsersCountry` function and simply return the closest utility.

Reply:
On reflection, it might be worth showing Tepco if the user's location is greater than z away from any utility (thereby establishing them as *probably* in another country without needing the Google API)
Though, its a bit of a stretch, so I'm happy either way 👍

Reply:
Hey @FraserTooth , I have modified the code to not use the Geocoding API. Instead, I have made use of a kind of Geocoordinate box around Japan to figure out if the user is probably in Japan or not. Please take a look and let me know. Also, the preview deployment is giving a CORS error on the page load. I tested the code in my local. 


Also, just another reminder:
> Could you add the topic `hacktoberfest` to your repository or add `hacktoberfest-accepted` label just to opt this PR in for Hacktoberfest? Thank you!

Reply:
Thanks for the idea of using lat, long values to check the boundaries of the user.

Reply:
Yeah the CORS issue was because I've updated the endpoint. If you'd pulled master it would've fixed it 👍

Reply:
oh my bad, did not even think that, got it 👍